types/dragselect/index.d.ts

```diff
@@ -0,0 +1,302 @@
+// Type definitions for DragSelect 1.13
+// Project: https://ThibaultJanBeyer.com/
+// Definitions by: IceCreamYou <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace DragSelect;
```

Comment:
So this means that in a global context, you'd have to write `new DragSelect.DragSelect`, which seems wrong. You might want to say `export = DragSelect`, and remove the `export` modifier from the class.

Reply:
I removed the `export as namespace` because you're clearly right about that - `new DragSelect.DragSelect` is not what I meant to express. However, I'm unclear why you're suggesting `export =` could be a solution. In a file with imports/exports, I don't think that would make a difference other than changing the import syntax from `import { DragSelect } from 'dragselect'` to `import DragSelect = require('dragselect')`. In a file with no imports/exports, I don't think `export =` would make the class available as a global, would it? Is there a better way to export a (constructable) class as "available to import and also maybe global"?

Reply:
`export as namespace DragSelect` makes the whole module available in a script context as the global `DragSelect`. If the whole module _is_ the class declaration (doable with an `export =`), then `new DragSelect` in a global context will work. Also, [looking at it,](https://runkit.com/embed/f35hmlvj2oxl) it seems like `import DragSelect = require("dragselect")` is the _correct_ way to import it.

Reply:
Thanks for responding. I updated the PR as you suggested.